tuna/machine.py

```diff
@@ -251,7 +251,7 @@ def parse_agents(self) -> dict:  #pylint: disable=too-many-locals
             sub = stack.pop()
 
         if ':' in decoded_line:
-          cols = decoded_line.split(':')
+          cols = decoded_line.split(':', 1)
```

Comment:
Full arch string is something like "gfx90a:sramecc+:xnack-" and we only want the first part.